python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 try:
-    from eups import *
+    from eups import *  # noqa F403 F401 Importing everything into this namespace
```

Comment:
I suppose this is so all eups symbols come in when this module is imported by other code?

If it's just for local usage it'd be better to change this.